sdk/maps/azure-maps-elevation/src/main/java/com/azure/maps/elevation/implementation/models/LatLongPairAbbreviated.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.maps.elevation.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** A location represented as a latitude and longitude using short names 'lat' &amp; 'lon'. */
+@Fluent
+public final class LatLongPairAbbreviated {
```

Comment:
why do we need this model? Can we just use @JsonAlias with multiple names? 
consider bringing this feedback to the service: Do they really need two identical models, but one with short property names?

Reply:
This model is hidden in implementations so it is not open to clients using this API - it is wrapped with GeoPosition from Azure Core instead

Reply:
I mean from the implementation stand-point and service API standpoint - it seems service supports two very similar APIs. It's a feedback to the service, not the SDK

Reply:
To change this would be a change to the service side swagger which I believe is not changeable anymore according to my team. This is why we are wrapping it with other types to hide it from the user.

Reply:
my understanding that API is still in [preview](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/maps/data-plane/DEM/preview/1.0). Anyway, this does not change Java API surface and does not block this PR, but I still kindly suggest to clean it up, while API is not stable.